.circleci/config.yml

```diff
@@ -317,7 +330,7 @@ jobs:
       - run:
           name: Sync files to code.angularjs.org
           command: |
-            gsutil -m rsync -r ${PROJECT_ROOT}/deploy/code gs://code-angularjs-org-338b8.appspot.com
+            gsutil -m rsync -r ~/ng/deploy/code gs://code-angularjs-org-338b8.appspot.com
```

Comment:
For some reason `${PROJECT_ROOT}` was evaluating to `''`, so that approach was not working.